tests/trainer/connectors/test_accelerator_connector.py

```diff
@@ -0,0 +1,30 @@
+import torch
+
+from pytorch_lightning import Trainer
+from pytorch_lightning.accelerators import Accelerator
+from pytorch_lightning.plugins import DDPPlugin, PrecisionPlugin, SingleDevicePlugin
+
+
+def test_accelerator_training_type_plugin():
```

Comment:
can you move this test to `tests/accelerators/test_accelerator_connector` and integrate it with `test_custom_accelerator`?

Reply:
done